src/cli/bunx_command.zig

```diff
@@ -402,16 +485,16 @@ pub const BunxCommand = struct {
                             absolute_in_cache_dir,
                         )) |destination| {
                             const out = bun.asByteSlice(destination);
-                            _ = try Run.runBinary(
+                            try Run.runBinary(
                                 ctx,
                                 try this_bundler.fs.dirname_store.append(@TypeOf(out), out),
                                 this_bundler.fs.top_level_dir,
                                 this_bundler.env,
                                 passthrough,
                                 null,
                             );
-                            // we are done!
-                            Global.exit(0);
+                            // runBinary is noreturn
+                            comptime unreachable;
```

Comment:
maybe we should give it `!noreturn` return type. It looks like all the switch cases are handled